test/bootstrap.wetland.test.js

```diff
@@ -0,0 +1,52 @@
+    sails      = require('sails');
+var Barrels    = require('barrels');
+var path       = require('path');
```

Comment:
`const`?

Reply:
It was the only way I got it working.
With const it is also failing -> 

debug:  • https://sailsjs.com/support
debug: 
error: A hook (`policies`) failed to load!
  1) "before all" hook
error: A hook (`orm`) failed to load!

  0 passing (2s)
  1 failing

  1)  "before all" hook:
     ReferenceError: sails is not defined
      at Hook.<anonymous> (lib/loaderOverride.js:16:18)
      at Hook.wrapper [as loadPolicies] (node_modules/lodash/lodash.js:4968:19)
      at Hook.loadMiddleware (node_modules/sails/lib/hooks/policies/index.js:65:21)
      at Hook.wrapper [as loadMiddleware] (node_modules/@sailshq/lodash/lib/index.js:3250:19)
      at Hook.initialize (node_modules/sai

Reply:
But honestly ... I don't understand why